lib/src/testing/utils/testrunner.dart

```diff
@@ -809,7 +809,7 @@ void _runTests<PrivateKey, PublicKey>(
       signature = c.signature;
     } else {
       test('create signature', () async {
-        signature = await r._signBytes!(
+        signature = await r._signBytes(
```

Comment:
Why these changes? Is it perhaps because we updated the language version when we bumped the SDK constraint? (Just guessing that maybe this had an impact on inference for nullability)

If so, can we consider bumping the SDK constraint in a differnet PR?

Reply:
As you may have guessed, this is what lint pointed out because of the newer version of dart. The following PR separates the Flutter and Dart version upgrades and `dart fix --apply`.

https://github.com/google/webcrypto.dart/pull/87